src/plugins/intel_cpu/tests/functional/subgraph_tests/src/mha.cpp

```diff
@@ -187,18 +187,23 @@ class MHATest : public testing::WithParamInterface<MHATuple>,
         for (int i = 0; i < funcInputs.size(); ++i) {
             const auto& funcInput = funcInputs[i];
             ov::Tensor tensor;
-            tensor = ov::test::utils::create_and_fill_tensor_normal_distribution(funcInput.get_element_type(), targetInputStaticShapes[i], 1.0f, 0.5f);
+            // TODO: after snippets fixed should remove 2nd condition
```

Comment:
Could you create the ticket for Snippets and assign to me? It'd be great if the ticket number is in comment. Thank you!

Reply:
Done, ticket: 105339